<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="in.biencaps.erp.api.utilities.SuiteListener"/>
  </listeners>
  <test thread-count="5" name="Test(failed)">
    <classes>
      <class name="in.biencaps.erp.api.testcases.DepartmentFolderAPITestCases">
        <methods>
          <include name="verify_Get_Employees_By_Department_Name_With_Authorization"/>
          <include name="extent_Report_Setup"/>
          <include name="startBackendServer"/>
          <include name="flush_Extent_Report"/>
          <include name="test_Method_Exection_Result"/>
        </methods>
      </class> <!-- in.biencaps.erp.api.testcases.DepartmentFolderAPITestCases -->
      <class name="in.biencaps.erp.api.testcases.ProjectFolderAPITestCases">
        <methods>
          <include name="verify_Add_Project_With_Authorization" invocation-numbers="1 3 6 8 9 10 11"/>
          <include name="extent_Report_Setup"/>
          <include name="startBackendServer"/>
          <include name="verify_Delete_Project_With_Authorization" invocation-numbers="2"/>
          <include name="flush_Extent_Report"/>
          <include name="test_Method_Exection_Result"/>
        </methods>
      </class> <!-- in.biencaps.erp.api.testcases.ProjectFolderAPITestCases -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
